src/systems/filecoin_markets/storage_market/storage_market_actor.go

```diff
@@ -0,0 +1,113 @@
+package storage_market
+
+import addr "github.com/filecoin-project/specs/systems/filecoin_vm/actor/address"
+import deal "github.com/filecoin-project/specs/systems/filecoin_markets/deal"
+import actor "github.com/filecoin-project/specs/systems/filecoin_vm/actor"
+
+func (sma *StorageMarketActor_I) WithdrawBalance(balance actor.TokenAmount) {
```

Comment:
actor code is tricky, see `InitActorCode` for how to do this. (runtime, state, etc)